sdk/keyvault/keyvault-certificates/samples/issuers.ts

```diff
@@ -42,22 +42,14 @@ async function main(): Promise<void> {
 
   // Reading the certificate will give us back the issuer name, but no other information.
   const certificate = await client.getCertificate("MyCertificate", "");
```

Comment:
This looks like it still gives the error: "Error: The 'version' cannot be empty."